python-sdk/src/astro/databases/base.py

```diff
@@ -183,6 +183,21 @@ def default_metadata(self) -> Metadata:
         """
         raise NotImplementedError
 
+    def openlineage_dataset_namespace(self) -> str:
+        """
+        Returns the open lineage dataset namespace as per
+        https://github.com/OpenLineage/OpenLineage/blob/main/spec/Naming.md
+        """
+        airflow_conn = self.hook.get_connection(self.conn_id)
+        return f"{self.sql_type}://{airflow_conn.host}"
```

Comment:
just thinking the host param might be null for some type of connection. for example in biquery at least in the airflow provider hook, we do not have a param host. I'm not sure if airflow internally keeps it in db based on other params of bigquery hook so does the connection object. We should need to test this flow. WDYT

Reply:
I have added the test here https://github.com/astronomer/astro-sdk/pull/1025/commits/fa62c625f9532727de2bc715eeec66db677208d6